docs/concepts.rst

```diff
@@ -103,3 +103,46 @@ See also
 
 * :term:`Kinto glossary <CRUD>`, for formal definitions of terms used in the documentation
 * :blog:`Our blog post <en/handling-permissions>` on permissions design in *Kinto*
+
+
+.. _technical-architecture:
+
+Technical Architecture
+======================
+
+Every concept of bucket, group, collection or record leverages the REST *resource*
+notion of *Cliquet*.
+
+*Cliquet* is a toolkit which brings abstractions and
+good practices to build HTTP microservices, such as data-driven REST APIs.
+
+*Cliquet* is relies on *Cornice* and *Pyramid* to implement the endpoints
+and HTTP request/response handling.
+
+.. image:: images/architecture.svg
```

Comment:
Do you plan on adding "comments" like you did below for the sequence diagrams? Btw, are those "comments" visible somewhere in the final page (in the "alt" of the image?)